### ISSUE TITLE
Cannot access `cookies` in graphql resolver

### ISSUE THREAD

**Issue by usaid-hasan** [kettanaito/msw#1852]
### Prerequisites

- [X] I confirm my issue is not in the [opened issues](https://github.com/mswjs/msw/issues)
- [X] I confirm the [Frequently Asked Questions](https://mswjs.io/docs/faq) didn't contain the answer to my issue

### Environment check

- [X] I'm using the [latest](https://github.com/mswjs/msw/releases/latest) `msw` version
- [X] I'm using Node.js version 14 or higher

### Browsers

Chromium (Chrome, Brave, etc.), Firefox

### Reproduction repository

https://github.com/usaid-hasan/msw-bug-reprod

### Reproduction steps

- npm install
- npm run dev

### Current behavior

Currently `cookies` is only available inside http resolvers but not in graphql. Also when trying to access cookie from `request.headers.get('cookie')`, also returns `null` both in http and graphql resolvers.

```javascript
// handlers.js
import { HttpResponse, graphql, http } from 'msw';

export default [
  graphql.query('GetUser', async ({ cookies, request }) => {
    console.log(cookies); // undefined
    console.log(request.headers.get('cookie')); // null

    if (!cookies?.jwt) {
      return HttpResponse.json({ errors: [{ message: 'You need to be signed in to access this' }]});
    }

    return HttpResponse.json({ data: { 
      user: {
        id: '507f1f77bcf86cd799439011',
        username: 'test_user',
        email: 'test_email@abc.com'
      }
    }});
  }),
];
```

### Expected behavior

`cookies` should be available in both http and graphql resolvers.

**Comment by kettanaito** [comment]
Thanks for reporting this, @usaid-hasan! I"m fixing this in #1853.

**Comment by kettanaito** [comment]
## Released: v2.0.6 🎉

This has been released in v2.0.6!

- 📄 [**Release notes**](https://github.com/mswjs/msw/releases/tag/v2.0.6)
- 📦 [npm package](https://www.npmjs.com/package/msw/v/2.0.6)

Make sure to always update to the latest version (`npm i msw@latest`) to get the newest features and bug fixes.

---

_Predictable release automation by [@ossjs/release](https://github.com/ossjs/release)_.
